### PR TITLE
fix typos in TARGET_LINK_LIBRARIES

### PR DESCRIPTION
**Pull request by mt5555**
Fixes issue #1064

### CODE REVIEW

File: pio/CMakeLists.txt [hunk 1]
```diff
@@ -175,12 +175,8 @@ if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL "GNU")
 endif()
 
 ADD_LIBRARY(pio ${SRCS_F90} ${SRCS_C} ${TEMPSRCF90})
-if(PNETCDF_LIBRARY)
-  TARGET_LINK_LIBRARIES(pio ${PNETCDF_LIBRARIES})
-ENDIF()
-if(NETCDF_LIBRARIES)
-  TARGET_LINK_LIBRARIES(pio ${NETCDF_LIBRARIES})
-ENDIF()
+TARGET_LINK_LIBRARIES(pio ${PnetCDF_Fortran_LIBRARIES})
```
Comment (jayeshkrishna) [inline pio/CMakeLists.txt]: Can you add the conditionals (if(PnetCDF_Fortran_FOUND) and if(NetCDF_Fortran_FOUND) should work I think) back? PIO1 can be built without netcdf or pnetcdf libs (although i don't try it a binary format is supported by pio1).

**Comment by jayeshkrishna** [comment]
@jedwards4b : The conditionals are still required since users can built with different combinations of netcdf, pnetcdf libs (and neither netcdf or pnetcdf)

**Comment by jedwards4b** [comment]
According to MT ```Also, I dont think the if() is needed, since if they are not set TARGET_LINK_LIBRARIES will ignore them?```  I assume he tested that?

**Comment by jayeshkrishna** [comment]
ok, cmake magic :)

**Comment by mt5555** [comment]
Yes, I verified the if() is not needed.  

if findPnetCDF.cmake doens't find pnetcdf, the variable is unset and TARGET_LINK_LIBRARIES is smart enough to ignore it.

**Pull request merged**